DependencyInjection/Configuration.php

```diff
@@ -477,6 +478,7 @@ public function getConfigTreeBuilder()
                             ->scalarNode('title')->defaultNull()->end() // pushover
                             ->scalarNode('host')->defaultNull()->end() // syslogudp & hipchat
                             ->scalarNode('port')->defaultValue(514)->end() // syslogudp
+                            ->integerNode('rfc')->defaultValue(1)->end() // syslogudp
```

Comment:
I think this should be an enumNode and list the possible values. Also please use the constants from \Monolog\Handler\SyslogUdpHandler instead of plain int.